context.go

```diff
@@ -63,9 +69,9 @@ type Context[H Hash] struct {
 	ChangeViewPayloads []ConsensusPayload[H]
 	// LastChangeViewPayloads stores consensus ChangeView payloads for the last epoch.
 	LastChangeViewPayloads []ConsensusPayload[H]
-	// LastSeenMessage array stores the height of the last seen message, for each validator.
-	// if this node never heard from validator i, LastSeenMessage[i] will be -1.
-	LastSeenMessage []*HV
+	// LastSeenMessage array stores the height and view of the last seen message, for each validator.
+	// if this node never heard from validator i, LastSeenMessage[i] will be nil.
```

Comment:
New sentence, capitalize the first word.

Reply:
Fixed.